docs/Usage.md

```diff
@@ -0,0 +1,323 @@
+
+3. **Create a Kubernetes Secret (by the Kubernetes Administrator)**
+
+   The Kubernetes Administrator creates a secret containing the above credentials and configuration details:
+
+   ```bash
+   cat <<EOF | kubectl apply -f -
+   apiVersion: v1
+   kind: Secret
+   metadata:
+     name: s3-secret-for-cosi
+   type: Opaque
+   stringData:
+     accessKeyId: <ACCESS_KEY_ID>
+     secretAccessKey: <SECRET_ACCESS_KEY>
+     endpoint: <S3_ENDPOINT>
+     region: <REGION>
+     iamEndpoint: <OPTIONAL_IAM_ENDPOINT>
+     tlsCert: |-
+       -----BEGIN CERTIFICATE-----
+       ...
+       -----END CERTIFICATE-----
+   EOF
+   ```
+
+   > **Note**
+   > Update `<ACCESS_KEY_ID>`, `<SECRET_ACCESS_KEY>`, `<S3_ENDPOINT>`, `<REGION>`, with valid values for your environment. If your endpoint does not require a TLS cert, you can remove it. Similarly, add IAM endpoint with `iamEndpoint` if its different from S3 endpoint otherwise remove it.
+   > If using TLS cert, include the certificate content (PEM-encoded) in the stringData section of the Secret. Use a multi-line block scalar (|-) in YAML so that the certificate (with newlines) is preserved correctly.
+
+---
+
+## 1. Bucket Provisioning
+
+In the **Scality COSI Driver**, both **Greenfield** and **Brownfield** provisioning share similar steps, with minor differences in how resources (Bucket, BucketClaim) are created.
+
+> Note:
+> For **fully working** examples, see the YAMLs in the [cosi-examples/brownfield](./cosi-examples/brownfield/) and [cosi-examples/greenfield](./cosi-examples/greenfield/) directories.
+> For brownfield scenario it is madatory to create COSI CRs in the same namespace as COSI driver and controller.
+
+### 1.1 Greenfield: Creating a New Bucket
+
+Greenfield provisioning will create a brand-new S3 bucket in your object store, managed by Kubernetes. Examples can be found [here](../cosi-examples/greenfield/).
+
+1. **Create a BucketClass**
+   A `BucketClass` defines how buckets should be provisioned or deleted. The bucket class name is used as a prefix for bucket name by COSI:
+
+   ```bash
+   cat <<EOF | kubectl apply -f -
+   apiVersion: objectstorage.k8s.io/v1alpha1
+   kind: BucketClass
+   metadata:
+     name: greenfield-bucketclass
+     namespace: container-object-storage-system
+   driverName: cosi.scality.com
+   deletionPolicy: Delete
+   parameters:
+     objectStorageSecretName: s3-secret-for-cosi
+     objectStorageSecretNamespace: default
+   EOF
+   ```
+
+   - `driverName` must match the Scality driver (default: `cosi.scality.com`). This determined by driver-prefix in the COSI driver deployment. For more information check [COSI Driver Parameters](./docs/driver-params.md).
+   - `deletionPolicy` can be `Delete` or `Retain`.
+   - `objectStorageSecretName` and `objectStorageSecretNamespace` reference the secret you created earlier in [Common Setup Steps](#common-setup-steps). For more information check [COSI Driver Parameters](./docs/driver-params.md).
```

Comment:
404

```suggestion
   - `driverName` must match the Scality driver (default: `cosi.scality.com`). This determined by driver-prefix in the COSI driver deployment. For more information check [COSI Driver Parameters](./driver-params.md).
   - `deletionPolicy` can be `Delete` or `Retain`.
   - `objectStorageSecretName` and `objectStorageSecretNamespace` reference the secret you created earlier in [Common Setup Steps](#common-setup-steps). For more information check [COSI Driver Parameters](./driver-params.md).
```

Reply:
THanks, I moved the location, corrected.